client/src/components/Messaging/Emoji/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './Style.module.css';
+
+const Emoji = ({ symbol, text, setText }) => {
+  const changeText = () => {
+    setText(text + symbol);
+  };
+  return (
+    <span className={styles.emoji} role="img" onClick={changeText}>
```

Comment:
```suggestion
const Emoji = ({ symbol, onClick }) => {
  return (
    <span className={styles.emoji} role="img" onClick={onClick}>
```

Reply:
Symbol is passed from EmojiRow to Emoji and changeText requires symbol to change the text.